Resources/Prototypes/DeltaV/Body/Organs/feroxi.yml

```diff
@@ -0,0 +1,58 @@
+- type: entity
+  id: OrganFeroxiStomach
+  parent: OrganAnimalStomach
+  categories: [ HideSpawnMenu ]
+  components:
```

Comment:
COppied the order from another one, go fix it :blunt:

Reply:
Also its not consistent in ss14 so there is no convention in ss14

Reply:
> Also its not consistent in ss14 so there is no convention in ss14

https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions
just because people people put it randomly when making their shitcode 2 years ago doesnt mean there's no conventions now